web/src/dark_theme.ts

```diff
@@ -22,9 +24,17 @@ export function disable(): void {
         const ls = localstorage();
         ls.set("spectator-theme-preference", "light");
         user_settings.color_scheme = settings_config.color_scheme_values.day.code;
+        realm_logo.render();
     }
 }
 
 export function default_preference_checker(): void {
```

Comment:
Should we rename this function and library? `dark_theme.default_preference_checker` is a weird thing to call.

I would prefer it to be called `theme.use_system_default` instead.

Reply:
Let's open a czo thread to weight in opinion from others.

Reply:
Opened a [CZO thread](https://chat.zulip.org/#narrow/stream/6-frontend/topic/refactor.20dark_theme.2Ets.20module/near/1821507) discussing this idea, thanks!